transit_odp/publish/forms/seasonal_services.py

```diff
@@ -64,6 +70,7 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         self.org_id = kwargs.pop("org_id", None)
+        self.page = kwargs.pop("page", None)
```

Comment:
we dont need this

Reply:
I think I need it

```
transit_odp/publish/templates/publish/seasonal_services/index.html:83->95

            <div class="govuk-width-container">
                <div class="govuk-grid-row">
                    <div class="govuk-grid-column-two-thirds">
                        <a role="button" class="govuk-button govuk-button--secondary govuk-!-margin-top-5" href="{% url 'add-seasonal-service' pk1=pk1 host hosts.publish %}?page={{ page_obj.number }}&">
                            {{ _('Add new date range') }}
                        </a>
                        <a role="button" class="govuk-button govuk-button--secondary govuk-!-margin-top-5" href="{% url 'feed-list' pk1=pk1 host hosts.publish %}">
                            {{ _('Close') }}
                        </a>
                    </div>
                </div>
            </div>
```

This is  the index.html file when you press the add-new-date button you need to know where the "consumer" pressed the button because at the end of the all process the consumer would likes to jump back where it was.
Also suppose someone starts the process to add new date and then refused to continue, it press the "cancel" button. Again he\she wants back where it was not at the first page!!

This page code\bug if you want I can leave or remove, but If I'll remove it I can't remove part of the code I have to remove everywhere.

Reply:
I really dont think it matters for edit and create. Storing state in query params feels weird.
I wouldnt expect to be returned to the same page.

You do make a great point for delete. We dont want the user to changes pages when they delete.